README.md

```diff
@@ -158,6 +158,16 @@ We support the following conditions over telemetry attributes and their values:
 
 ##### Tags
 
+When synthesizing an entitym we add some tags to it by default based on telemetry attributes and others based on the entity synthesis rules.
+
+###### Default tags
```

Comment:
Mention that some are reserved? (e.g. name, accountId, accountName)?

Reply:
Sure, I wasn't sure if we wanted to add them already (I'll add them in another PR though, so I can merge this one to try and re-trigger the bot).